packages/p2p/src/components/orders/order-table/order-table-content.jsx

```diff
@@ -45,16 +46,32 @@ const OrderTableContent = observer(() => {
 
         if (modified_list.length) {
             return (
-                <OrderTableHeader>
-                    <InfiniteDataList
-                        data_list_className='orders__data-list'
-                        has_more_items_to_load={order_store.has_more_items_to_load}
-                        items={modified_list}
-                        keyMapperFn={item => item.id}
-                        loadMoreRowsFn={order_store.loadMoreOrders}
-                        rowRenderer={row_props => <OrderRow {...row_props} />}
-                    />
-                </OrderTableHeader>
+                <React.Fragment>
```

Comment:
You can get rid of this fragment and avoid ternary statements by creating an if statement instead, e.g.:

```JS
if (isMobile()) {
    return (
        <MobileComponent />
    );
}

return (
    <DesktopComponent />
);
```